src/lib.rs

```diff
@@ -842,15 +842,15 @@ impl<T: TensorType> Tensor<T> {
     }
 
     /// Sets (copies) the tensor values to the provided ones.
-    /// 
+    ///
     /// ```
     /// # use tensorflow::Tensor;
     /// let a = Tensor::new(&[2, 2]).with_values(&[0_i32, 1, 2, 3]).unwrap();
     /// ```
     pub fn with_values(mut self, value: &[T]) -> Result<Self> {
-        if self.len() != value.len() { 
+        if self.len() != value.len() {
             return Err(invalid_arg!(
-                "length of values array ({}) is not equal to tensor total elements ({})", 
```

Comment:
The trimmed trailing white-space was simply due to editor save. Seems to be solid though.